### DISCUSSION TITLE
Use state-props in an offline app based on Tiddlywiki

### DISCUSSION THREAD

**Discussion by kookma** [argyleink/open-props#84] [Q&A] [answered]
Thanks a millions! open-probs is amazing!

I like to use open-props in my own [Tiddlywiki](https://tiddlywiki.com/prerelease)  here: TW-Shiraz (https://kookma.github.io/TW-Shiraz/)
It uses a selection of bootstrap 4 right now! Tiddlywiki does not allow to include JS directly and needs to develop a plugin to do so!

My question is can I have much of the open-props feature just by importing the css?

**Comment by argyleink** [comment, marked as answer]
yep, you can "just import them" from a cdn 🙂  no js, npm or build step required

import all of them with `https://unpkg.com/open-props`
or some of them with `https://unpkg.com/open-props/colors.min.css`

**Comment by kookma** [comment]
Thanks a million @argyleink 
I will share the Open-Props-Tiddlywiki once I made it!
